### PR TITLE
added comment to noxfile and added suggestion to README

### PR DESCRIPTION
**Pull request by PhilipSatterthwaite**
Pull request for APC 524 class assignment.
- added a comment in the noxfile
- added 2 suggested sections for README

### CODE REVIEW

File: noxfile.py [hunk 1]
```diff
@@ -11,6 +11,9 @@ def tests(session: nox.Session) -> None:
 
 @nox.session(python="3.11")
 def lint(session):
+    """
+    Run the linter.
```
Comment (PhilipSatterthwaite) [inline noxfile.py:15]:
```suggestion
    Run the flake8 linter.
```

File: README.md [hunk 2]
```diff
@@ -41,3 +41,11 @@ The solver implemented in `FIEFS` is a MUSCL-Hancock scheme as described in [1].
 
 [1] Toro, E. F. (2011). Riemann solvers and Numerical Methods for fluid dynamics: A practical introduction. Springer.
 [2] Boerchers, J., Rzepka, S., and Fush, T. (2022). PSYCHo-I - Python Simulations Yielding Hydrodynamic Instabilities. Github repository: `https://github.com/johnboerchers/psycho-i.`
+
+##List of original files (created by Michael and Philip)
+noxfile.py
+pyproject.toml
+.pre-commit-config.yaml
+
+##List of modified files (and modifications)
```
Comment (PhilipSatterthwaite) [inline README.md:50]: I believe there are some files missing here. Other files had modifications done to them such as the input files.

**Review by PhilipSatterthwaite — COMMENTED** [review]
Overall looks good. Comment on the linter in noxfile.py could be more specific, and list of modified files in README.md does not seem complete.

File: README.md (hunk 2 above)
Comment (mw6136) [reply README.md:50]: I have updated the citations, as discussed. Indicating the original files.

File: noxfile.py [hunk 3]
```diff
@@ -11,6 +11,9 @@ def tests(session: nox.Session) -> None:
 
 @nox.session(python="3.11")
```
Comment (mw6136) [inline noxfile.py:12]: Similarly tests should capture Python 3.8-3.11 at least, I believe it is sufficient to call nox only in 3.11 though.

**Pull request closed without merging**